differential_drive_ekf_node:
  ros__parameters:
    initial_state: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    initial_covariance: [1.0, 0.0, 0.0, 0.0, 0.0, 0.0,
                         0.0, 1.0, 0.0, 0.0, 0.0, 0.0,
                         0.0, 0.0, 0.1, 0.0, 0.0, 0.0,
                         0.0, 0.0, 0.0, 0.1, 0.0, 0.0,
                         0.0, 0.0, 0.0, 0.0, 0.1, 0.0,
                         0.0, 0.0, 0.0, 0.0, 0.0, 0.1]
    process_noise_covariance: [0.1, 0.0, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.1, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.01, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.1, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.1, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.0, 0.01]
    measurement_noise_covariance: [0.1, 0.0, 0.0, 0.0,
                                   0.0, 0.1, 0.0, 0.0,
                                   0.0, 0.0, 0.01, 0.0,
                                   0.0, 0.0, 0.0, 0.005]
    wheel_base: 0.5
    wheel_radius: 0.1
    update_rate: 50.0